velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -998,5 +998,17 @@ VELOX_INSTANTIATE_TEST_SUITE_P(
     MinMaxByGroupByAggregationTest,
     testing::ValuesIn(getTestParams()));
 
+TEST_F(MinMaxByGroupByAggregationTest, largeValuesDate) {
```

Comment:
Like we discussed we should change this to use a similar framework for the other type testing in this file.

Changing valueAt for Date to return the corresponding string for date should suffice.